demo/validate.php

```
@@ -0,0 +1,76 @@
+ */
+function countLogEntries(array $node): int
+{
+    $count = 0;
+    $opens = $node['open'] ?? [];
+    if (is_array($opens)) {
+        foreach ($opens as $child) {
+            if (! is_array($child)) {
+                continue;
+            }
+
+            $count += 1 + countLogEntries($child); // the scope itself + its contents
```

PHPStan (2.2.7)

[error] Parameter `#1` $node of function countLogEntries expects array<string, mixed>, array<mixed, mixed> given.


(argument.type)